packages/select/src/FilterMultiSelect/components/ListBoxSection/ListBoxSection.tsx

```diff
@@ -1,24 +1,52 @@
-import React from "react"
+import React, { ReactNode } from "react"
+import { v4 } from "uuid"
 import { MultiSelectItem } from "../../../types"
 import styles from "./ListBoxSection.module.scss"
-
 export interface ListBoxSectionProps {
   items: MultiSelectItem[]
   /**
    * Becomes an aria-label on the section, informing
    * unsighted users
    */
   sectionName: string
+  /** * If provided, will override the aria-label for this group */
+  sectionHeader?: ReactNode
   children: (item: MultiSelectItem) => React.ReactNode
 }
 
 export const ListBoxSection = ({
   items,
   sectionName,
   children,
-}: ListBoxSectionProps): JSX.Element => (
-  <div aria-label={sectionName} className={styles.listBoxSection}>
-    {Array.from(items).map(node => node != undefined && children(node))}
-  </div>
-)
+  sectionHeader,
+}: ListBoxSectionProps): JSX.Element => {
+  const listSectionId = sectionHeader ? v4() : undefined
+  return (
+    <li role="presentation">
+      <ul
+        className={styles.listBoxSection}
+        aria-label={sectionHeader ? undefined : sectionName}
```

Comment:
I don't think this is a good idea, unless you enforce it with types, cause if I put ` sectionName` and a `sectionHeader` it's not obvious I'd lose functionality in one of these props. I can also imagine someone using the `aria-label` for testing...

Reply:
This is true, only problem is we don't want to have both aria-label and aria-lablledby - could potentially use aria-labelled and have both in that section with a visually hidden section title

Reply:
Though the problem is this adds hidden items into the DOM which increase the listbox counter for how many children it has 🤔

Reply:
Think I've found a solution that works, its still isn't read well with the screen reader (at least on osx) since we have to use role presentation, but this should should always have the sectionName as the accessible title for a group, otherwise will combine it with the sectionHeader if a sectionHeader is given.

With sectionHeader
![Screenshot 2023-07-19 at 4 17 59 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/129b5a00-cd01-4f7a-9d0a-3569cb3d59c5)

Without sectionHeader
![Screenshot 2023-07-19 at 4 25 09 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/0f6ed2ba-ed48-43b5-b986-db946ec1945d)